src/SFA.DAS.AssessorService.Application.Api.UnitTests/Services/CertificateNameCapitalisationServiceTests.cs

```diff
@@ -0,0 +1,163 @@
+﻿using Moq;
+using NUnit.Framework;
+using SFA.DAS.AssessorService.Application.Api.Services;
+
+namespace SFA.DAS.AssessorService.Application.Api.UnitTests.Services
+{
+    [TestFixture]
+    public class CertificateNameCapitalisationServiceTests
+    {
+        private Mock<CertificateNameCapitalisationService> _certificateNameCapitalisationService;
```

Comment:
Since the service is being tested here you would instantiate the actual class not a Mock<T>